README.md

```diff
@@ -12,12 +12,13 @@ This allows users of [RWS LanguageCloud](https://www.rws.com/translation/languag
 ## Setup
 
 1. `pip install wagtail-localize-rws-languagecloud`
-2. Add to `INSTALLED_APPS` in Django settings:
+2. Add `wagtail_localize_rws_languagecloud` to `INSTALLED_APPS` _before_ `wagtail.admin`:
```

Comment:
Is this a new requirement, or just a clarification? If it is new, lets make sure we call this out in the changelog for the next release. It could be a breaking change if you don't already have this ordering.

Reply:
Good call. Yes it's new. It's required so that we can override the `locale_selector.html` template.